### PR TITLE
First test using the trusted getter

### PR DESCRIPTION
**Pull request by arieljbirnbaum**
First baby step towards #1682.

This step mimics the setup and the very first step of `identity.test.ts` but using the [trusted getter](https://github.com/litentry/litentry-parachain/pull/1732) to fetch the shielding key.

The same getter will also be needed to fetch the results of DI calls.

Includes a few small drive-by refactorings and fixes.

### CODE REVIEW

File: tee-worker/ts-tests/common/utils/context.ts
Comment (arieljbirnbaum) [inline tee-worker/ts-tests/common/utils/context.ts:1]: drive-by

File: tee-worker/ts-tests/identity-direct-invocation.test.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import { step } from 'mocha-steps';
+import { assert } from 'chai';
+import { hexToU8a } from '@polkadot/util';
+
+import { initIntegrationTestContext } from './common/utils';
+import {
+    createSignedTrustedGetterUserShieldingKey,
+    getTEEShieldingKey,
+    sendRequestFromTrustedGetter,
+} from './examples/direct-invocation/util'; // @fixme move to a better place
```
Comment (arieljbirnbaum) [inline tee-worker/ts-tests/identity-direct-invocation.test.ts:11]: These will be cleaned up with future progress on this test suite

File: tee-worker/ts-tests/identity-direct-invocation.test.ts [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import { IntegrationTestContext } from './common/type-definitions';
+
+describe('Test Identity (direct invocation)', function () {
+    let context: IntegrationTestContext = undefined as any;
+    let teeShieldingKey: KeyObject = undefined as any;
```
Comment (arieljbirnbaum) [inline tee-worker/ts-tests/identity-direct-invocation.test.ts:16]: Doing these here for now since I didn't want to mess with `describeLitentry` just yet; will figure a better way later. Maybe we can just use Jest instead of Mocha and then we'd have async support in `describe`?

File: tee-worker/ts-tests/identity-direct-invocation.test.ts [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+        // const resp_challengecode = await checkUserChallengeCode(
+        //     context,
+        //     'IdentityManagement',
+        //     'ChallengeCodes',
+        //     u8aToHex(context.substrateWallet.alice.addressRaw),
+        //     identity_hex
+        // );
+
+        // assert.equal(resp_challengecode, '0x', 'challengecode should be empty before create');
```
Comment (arieljbirnbaum) [inline tee-worker/ts-tests/identity-direct-invocation.test.ts:60]: This one's coming up next; I take it I'll have to start adding more trusted getter methods.

File: tee-worker/ts-tests/identity.test.ts [hunk 4]
```diff
@@ -701,7 +701,7 @@ describeLitentry('Test Identity', 0, (context) => {
         const [event] = await handleIdentityEvents(context, aesKey, resp_events, 'UserShieldingKeySet');
         await assertInitialIDGraphCreated(context.api, context.substrateWallet.eve, event);
 
-        let identities = [];
+        let identities: LitentryIdentity[] = [];
```
Comment (arieljbirnbaum) [inline tee-worker/ts-tests/identity.test.ts:704]: drive-by; IDE was complaining.

**Comment by Kailai-Wang** [comment]
CI failed tho with `error Command "test-identity-direct-invocation:staging" not found` 👀

**Comment by arieljbirnbaum** [comment]
> CI failed tho with `error Command "test-identity-direct-invocation:staging" not found` eyes

Yeah, forgot to enable it in the `package.json` :sweat_smile: Added now; let's see :crossed_fingers:

**Comment by arieljbirnbaum** [comment]
[There we go](https://github.com/litentry/litentry-parachain/actions/runs/5087902899/jobs/9143763178?pr=1735) :heavy_check_mark:

**Review by Kailai-Wang — APPROVED** [review]
Looks good, thanks!
